src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -251,6 +251,67 @@ class IndexManagerTests extends HyperspaceSuite with SQLHelper {
     }
   }
 
+  test("Verify refresh-incremental rebuild of index.") {
+    Seq(("csv", Map("header" -> "true")), ("parquet", Map()), ("json", Map())).foreach {
+      case (format, option: Map[String, String]) =>
+        spark.conf.set(IndexConstants.REFRESH_APPEND_ENABLED, true)
+        val refreshTestLocation = sampleParquetDataLocation + "refresh_" + format
```

Comment:
nit: for consistency:

```suggestion
        // Setup. Create sample data and index.
        val refreshTestLocation = sampleParquetDataLocation + "refresh_" + format
```